src/index.html

```diff
@@ -13,10 +13,21 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="stopwatch__hand">
+        <div
+          class="stopwatch__hand stopwatch__hand--min stopwatch__hand--min--speed-up"
+        ></div>
+        <div
+          class="stopwatch__hand stopwatch__hand--second stopwatch__hand--second--speed-up"
+        ></div>
```

Comment:
Same issue here with the class names. It's better to simplify them for better readability and maintainability of your code. For example, you can use `stopwatch__hand-second speed-up`.